src/http/index.ts

```diff
@@ -10,27 +10,38 @@ export interface Http {
 }
 
 export interface HttpOptions {
-  // an array of valid express ApplicationRequestHandlers (middlewares) injected BEFORE loading routes
-  preRouteApplicationRequestHandlers?: any[]
+  // Options injectable into the routes importer
+  routesImporter?: RoutesImporter,
+
+  // An array of valid express ApplicationRequestHandlers (middlewares) injected BEFORE loading routes
+  preRouteApplicationRequestHandlers?: [any | [string, any]]
+
   // an array of valid express ApplicationRequestHandlers (middlewares) injected AFTER loading routes
-  postRouteApplicationRequestHandlers?: any[]
+  postRouteApplicationRequestHandlers?: [any | [string, any]]
 }
 
 export default async (port: number, options?: HttpOptions): Promise<Http> => {
   const app = express();
+
+  const useRequestHandlers = (requestHandlers: [(...args: any) => any | [string, any]]) => {
+    requestHandlers.forEach((handler: any) => app.use(...handler));
```

Comment:
`requestHandlers?.forEach?.(`

will fail if options undefined

Reply:
actually no - the check is handled elsewhere, the function (as typed) expects an array to be passed